src/react-components/src/__tests__/components/Hero-test.tsx

```diff
@@ -0,0 +1,37 @@
+
+import { expect } from "chai";
+import { stub } from "sinon";
+import * as Enzyme from "enzyme";
+import * as React from "react";
+import * as Mocha from "mocha";
+
+import Heading from "../../components/01-atoms/Text/Headings/Heading";
+import { HeroTypes } from "../../components/03-organisms/Hero/HeroTypes";
+import Hero from "../../components/03-organisms/Hero/Hero";
+
+describe("Hero Test", () => {
+
+  let blurCallback;
+  let changeCallback;
+  let submitCallback;
+
+  beforeEach(() => {
+    changeCallback = stub();
+    blurCallback = stub();
+  });
+
+  it("Generates a Hero with a background image", () => {
```

Comment:
This test is good, but I'd have many more tests.  eg:  What happens if there's a HeroTypes.Primary but the user forgets to pass a background image? (which is optional)